src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawingAdapter.java

```diff
@@ -8,10 +8,10 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawingAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it lacks a driver aspect.
Inheritance is not used (no super drawLine calls). Should be removed!